platform/app/prisma/migrations/20260804120003_gateway_debits_process_name/migration.sql

```
@@ -0,0 +1,30 @@
+-- Collision-free by construction: nothing has ever registered under
+-- `gatewayDebits`, so no row can be updated onto an existing unique key. Each
+-- table's unique index leads with processName, which is exactly why the update
+-- is safe here and would not be if both names were in use.
+--
+-- One bounded gap remains, and it is accepted rather than solved: between this
+-- migration running and the last old-code pod draining, those pods keep writing
+-- rows under the old name, and any still pending when they stop are orphaned.
+-- That window is seconds of drain lag, and the ClickHouse ledger's
+-- ReplacingMergeTree key collapses anything written twice across the straddle.
+
+UPDATE "ProcessManagerInstance" SET "processName" = 'gatewayDebits' WHERE "processName" = 'attributedUserDebits';
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)